apps/ecs/views/ineligible-document.html

```diff
@@ -0,0 +1,10 @@
+{{<partials-page}}
+  {{$page-content}}
+
+  {{#t}}pages.ineligible-document.header{{/t}}
```

Comment:
Remove this line which is taken from header, as all these lines are added into the page content.